<!DOCTYPE html>
<html>
<head>
<link rel="stylesheet" href="style.css">
</head>
<body>
<!-- Side navigation -->
<div class="sidenav">
<a href="index.html">Home</a>
<a href="about.html">About</a>
<a href="contact.html">Contact</a>
</div>
<div class="main">
<h3> Has structural biology significantly impacted biology and is there a place for it in the 21st century? </h3>
<p>Structural biology aims to uncover the molecular architecture of life, the understanding of which has opened a new dimension into every field of biology. Since Robert Hooke’s observations of the cellular structure of cork under a light microscope, biologists have been seeking methods to analyse organic structures with greater resolution. The development of X-ray crystallography spurred the analysis of life in greater detail and allowed the discovery of the double-helix, which quickly explained many of the key principles of genetics (Thomas, 2012). Since, further methods of structural analysis have been developed including protein nuclear magnetic resonance spectroscopy (protein NMR) and cryogenic electron microscopy (Cryo-EM). Now, computational methods including artificial intelligence and machine learning are at the forefront of structural biology and rapidly accelerate it’s progress in the 21st century.
<p>Ernst Abbe (1873) determined that, due to the wavelength of visible light, the smallest biological structures theoretically observable under a microscope were around the size of a prokaryotic cell (1 µm). Eukaryotic cells and the general structure of their organelles are within reach, but light microscopy is unable to provide any insight into the cellular macromolecules, let alone their exact atomic construction (Curry, 2015). X-ray crystallography was the first technology to allow a closer look beyond these physical constraints. William and Lawrence Bragg were pioneers of this novel method of structural analysis, after deducing how to interpret the diffraction patterns captured by Max von Laue of copper sulphate crystals in 1912. X-ray crystallography soon became a vital technology in all three branches of the natural sciences (Thomas, 2012).
<p>In 1951, Rosalind Franklin and Maurice Wilkins captured high quality diffraction patterns of DNA and used them to make significant progress in calculating the macromolecule’s structure. Francis Crick and James Watson were, at the same time, attempting to decipher the organisation of DNA but lacking the crystallographic evidence. Franklin had this information, which demonstrated the arrangement of the sugar-phosphate backbone on the exterior of the double helix (National Library of Medicine, 2019). Via Wilkins, now estranged from Franklin, and Max Perutz, Watson and Crick obtained this data that would confirm their anti-parallel double-helix theory and award themselves and Wilkins the Nobel Prize for Medicine in 1962 (The Nobel Prize, 1962).
<p>The succinct paper detailing Watson and Crick’s proposed structure for DNA in Nature was not immediately successful. Amongst their findings, they noted that the complimentary base pairing system caused significant interest (Watson and Crick, 1953b). This promptly explained the replication mechanism of DNA, splaying the double helix and using either strand as a template, which was explained in more depth a month later (Watson and Crick, 1953a). Comparable to these revelations triggered by the structure of DNA being uncovered, the first tertiary protein structures spawned similar interest. John Kendrew et. al. (1958) deduced the structure of myoglobin using x-ray crystallography. The structure of the whale-derived myoglobin had folds similar to chains in horse-derived haemoglobin, suggesting an evolutionary link. 
<p>With time, techniques have been developed to further increase the resolution and ease of structural analysis. Cryo-EM has been developed since the 1970s and has become a strong competitor against x-ray crystallography (Callaway, 2020). The largest drawback to x-ray crystallography is the inherent need to crystallise the subject molecule, a process potentially taking years and limiting resolution based on purity. Cryo-EM is not straightforward, requiring that samples are frozen in vitreous ice, but eliminates a major hurdle (Wang and Wang, 2017). In terms of resolution, x-ray crystallography is still superior, with median resolutions on the Protein Data Bank (PDB) around double that of cryo-EM (Yip et al., 2020).
<p>The existence of these tools has permitted significant advances in the understanding of human disease, such as in the study of amyloid structures by Iadanza et. al. (2018) where cryo-EM and solid-state NMR were used. Prior to the existence of these technologies, this research would have been impossible. Drug discovery also relies heavily on structural analysis techniques whereby advanced understanding of disease-relevant factors enables novel methods for designing pharmacological compounds (Wendt et al., 2008). 
<p>Now, computational methods lead structural biology innovation. Vast quantities of data are generated from automated experimental procedures and are shared across the world (Nussinov, 2015). Repositories of data describing proteomes, genomes and transcriptomes permit the machine learning models to model biological systems in depth (Ozdemir et al., 2019), including the assembly of molecular pathways by inferring molecular interactions (Dimura et al., 2016). The advancement of processor architecture continues to rapidly increase available computational power for in silico simulation. A breakthrough was made by DeepMind in 2018 with their artificial intelligence powered protein structure prediction program AlphaFold 1. At CASP13, it performed with greater accuracy than around 50% of all other programs, excelling at predicting structures with no previously known template (Nussinov et al., 2019). Two years later, at CASP14, a vastly reworked AlphaFold 2 performed “astoundingly” well by making the most accurate prediction of 91% of the test targets (Heaven, W. D., 2020).
<p>In summary, structural biology has had a significant impact on biology. Without it, most if not all of modern drug discovery would have been impossible including therapies for cancers and HIV (Wendt et al., 2008). A revolution is occurring across all of biology, (Nussinov et al., 2019) whereby the key quest still remains: to observe and understand the underlying structure of life. Computational methods have transformed structural biology into the 21st century, and with the further development of technology, this paradigm shift may yield results greater than that of the first glimpses of DNA.
<hr>
<h3> References </h3>
<p>Abbe, E. (1873) ‘Beiträge zur Theorie des Mikroskops und der mikroskopischen Wahrnehmung’, Archiv für Mikroskopische Anatomie, 9(1), pp. 413–468. Available at: https://doi.org/10.1007/BF02956173.
<p>Callaway, E. (2020) ‘Revolutionary cryo-EM is taking over structural biology’, Nature, 578(7794), pp. 201–201. Available at: https://doi.org/10.1038/d41586-020-00341-9.
<p>Curry, S. (2015) ‘Structural Biology: A Century-long Journey into an Unseen World’, Interdisciplinary Science Reviews, 40(3), pp. 308–328. Available at: https://doi.org/10.1179/0308018815Z.000000000120.
<p>Dimura, M. et al. (2016) ‘Quantitative FRET studies and integrative modeling unravel the structure and dynamics of biomolecular systems’, Current Opinion in Structural Biology, 40, pp. 163–185. Available at: https://doi.org/10.1016/j.sbi.2016.11.012.
<p>Heaven, W. D. (2020) DeepMind’s protein-folding AI has solved a 50-year-old grand challenge of biology, MIT Technology Review. Available at: https://www.technologyreview.com/2020/11/30/1012712/deepmind-protein-folding-ai-solved-biology-science-drugs-disease/ (Accessed: 15 October 2022).
<p>Iadanza, M.G. et al. (2018) ‘A new era for understanding amyloid structures and disease’, Nature Reviews Molecular Cell Biology, 19(12), pp. 755–773. Available at: https://doi.org/10.1038/s41580-018-0060-8.
<p>Kendrew, J.C. et al. (1958) ‘A Three-Dimensional Model of the Myoglobin Molecule Obtained by X-Ray Analysis’, Nature, 181(4610), pp. 662–666. Available at: https://doi.org/10.1038/181662a0.
<p>National Library of Medicine (2019) The Discovery of the Double Helix, 1951-1953, Francis Crick - Profiles in Science. Available at: https://profiles.nlm.nih.gov/spotlight/sc/feature/doublehelix (Accessed: 15 October 2022).
<p>Nussinov, R. (2015) ‘Advancements and Challenges in Computational Biology’, PLOS Computational Biology, 11(1), p. e1004053. Available at: https://doi.org/10.1371/journal.pcbi.1004053.
<p>Nussinov, R. et al. (2019) ‘Computational Structural Biology: Successes, Future Directions, and Challenges’, Molecules, 24(3), p. 637. Available at: https://doi.org/10.3390/molecules24030637.
<p>Ozdemir, E.S. et al. (2019) ‘Developments in integrative modeling with dynamical interfaces’, Current Opinion in Structural Biology, 56, pp. 11–17. Available at: https://doi.org/10.1016/j.sbi.2018.10.007.
<p>The Nobel Prize (1962) The Nobel Prize in Physiology or Medicine 1962, NobelPrize.org. Available at: https://www.nobelprize.org/prizes/medicine/1962/summary/ (Accessed: 15 October 2022).
<p>Thomas, J.M. (2012) ‘The birth of X-ray crystallography’, Nature, 491(7423), pp. 186–187. Available at: https://doi.org/10.1038/491186a.
<p>Wang, H.-W. and Wang, J.-W. (2017) ‘How cryo-electron microscopy and X-ray crystallography complement each other’, Protein Science, 26(1), pp. 32–39. Available at: https://doi.org/10.1002/pro.3022.
<p>Watson, J.D. and Crick, F.H.C. (1953a) ‘Genetical Implications of the Structure of Deoxyribonucleic Acid’, Nature, 171(4361), pp. 964–967. Available at: https://doi.org/10.1038/171964b0.
<p>Watson, J.D. and Crick, F.H.C. (1953b) ‘Molecular Structure of Nucleic Acids: A Structure for Deoxyribose Nucleic Acid’, Nature, 171(4356), pp. 737–738. Available at: https://doi.org/10.1038/171737a0.
<p>Wendt, K.U. et al. (2008) ‘Structures and diseases’, Nature Structural & Molecular Biology, 15(2), pp. 117–120. Available at: https://doi.org/10.1038/nsmb0208-117.
<p>Yip, K.M. et al. (2020) ‘Atomic-resolution protein structure determination by cryo-EM’, Nature, 587(7832), pp. 157–161. Available at: https://doi.org/10.1038/s41586-020-2833-4.
<hr>
<p> &copy 2022 Daniel Cooper All Rights Reserved </p>
</div> 
<div class="vl"></div>
</body>
</html> 